install/windows/build.py

```diff
@@ -45,10 +47,9 @@ def build_odemisviewer_exe():
     run_command(cpy_command)
     run_command(pyi_command, "odemis")
 
-
-def build_delphiviewer_exe():
-    run_command(cpy_command)
-    run_command(pyi_command, "delphi")
+# def build_delphiviewer_exe():
+#     run_command(cpy_command)
+#     run_command(pyi_command, "delphi")
```

Comment:
Either udelete this function or add a comment why it should remain commented and not deleted